### ISSUE TITLE
CR should allow entry of and show details

### ISSUE THREAD

**Issue by ronaldtse** [glossarist/glossarist-desktop#54]
Today a CR only has a short hash, shows a picture and the date.

As a first step we can implement some of these.

The full harmonized IEC CR:

CR Item | Description
-- | --
Request ID: | Automatic counter added by the maintenance portal Prefix C followed by a sequence of 5 decimal digits in which leading zeroes are used but redundant, e.g. C00059
Request title | Free text; short title describing the intention of the CR (e.g. “To add 1 new REACH SVHC (Dibutylbis(pentane-2,4-dionato-O,O')tin) into DSL List”)
Applies to item(s): | Free textExplanation of the IEV items to which the Change request (CR) applies A CR can apply to one or several items in one or more IEV parts
Purpose of change request: | Picklist with values: Addition / Correction / Minor revision / Technical revision / Harmonization / Withdrawal Addition / Correction / Withdrawal: self explanatoryMinor revision: Editorial changes only to contentTechnical revision: Revision to technical contentHarmonization: when harmonizing different entries that refer to the same concept: e.g. "process" - 113-01-06, 192-01-08, 351-42-33, 904-01-05
Request process stage: | Picklist with values Proposal / Evaluation / Validation / Rejected / Withdrawn / Resolved / Extended procedure / Test Proposal: "Preparation for evaluation (Secretary) on Annex SL", Figure SL.1Evaluation / Validation / Rejected / Resolved / Extended procedure: As per Annex SLRejected / Withdrawn: As per "Annex SL, Figure SL.1" – but to date never used by TC 1Test: self explanatory Some have stage “Implemented” (i.e. “Resolved and available”)
ID (of existing entry): | ID of the database entry (e.g. DSL ID) if the CR is to change or delete an existing entry.  If new entry, then „TBD“ is used.
Proposal: | Free textDescription of the proposalSpecification of the process (Normal database procedure / Extended database procedure) recommended by the Terminology Coordinator/VT Convenor
Justification and explanation: | Free textJustification for the proposal, as well as any background information Supplementary information can be provided in a document uploaded as an attachment
Proposer: | Name of proposerEmail of proposerRole/title of proposerIEC 62474: also includes data field for National Committee (NC)
General comments: | Free text
Comments at evaluation: | Automated capture of the user login and date&time, plus some fixed text "Posted by cc-xx-bb on yyyy-mm-dd hh:mm:ss" followed byFree text
Voting at validation: | Automated capture of the user login and date&time, plus some fixed text "Posted by cc-xx-bb on yyyy-mm-dd hh:mm:ss" followed byFree textComment: Should be a picklist with values YES / NO / ABS for the vote, and free text field only for comments.
Comments at validation: | Automated capture of the user login and date&time, plus some fixed text "Posted by cc-xx-bb on yyyy-mm-dd hh:mm:ss" followed byFree text
Comment file upload: | Browse function to upload a file
Requested on: | “yyyy-mm-dd"
Evaluated on: | “yyyy-mm-dd"
Resolved on: | “yyyy-mm-dd"
Date Implemented: | “yyyy-mm-dd"
Withdrawn on: | “yyyy-mm-dd"
Evaluation closing on: | Free text,  but format "yyyy-mm-dd" is normaly usedFree text,Date on which the evaluation stage closes = 8 weeks after the evaluation start dateComment: Should be automatically calculated from the evaluation start date
Validation closing on: | Free text, but format "yyyy-mm-dd" is normaly usedFree text,Date on which the validation stage closes = 4 weeks after the validation start dateComment: Should be automatically calculated from the validation start date

**Comment by strogonoff** [comment]
@ronaldtse 

1. Does “harmonized” mean it applies to all users who make use of CR functionality?

2. What is the meaning of “Date Implemented”?

**Comment by ronaldtse** [comment]
> Does “harmonized” mean it applies to all users who make use of CR functionality?

Yes, it means that all CRs use the same structure. This is for IEC, we can tailor from this structure, of course. 

> What is the meaning of “Date Implemented”?

I think here it means the date the change is “Available” in the database. I’m not sure if it is useful for our cases...
